File: backend/config/environment.py
"""Validated environment settings."""

import os





EVENTS_TABLE = os.getenv("EVENTS_TABLE", "event-ticketing-events")
REGISTRATIONS_TABLE = os.getenv("REGISTRATIONS_TABLE", "event-ticketing-registrations")
EMAIL_INDEX = os.getenv("EMAIL_INDEX", "email-index")
REGISTRATION_ID_INDEX = os.getenv("REGISTRATION_ID_INDEX", "registration-id-index")
REGISTRATION_TOPIC_ARN = os.getenv("REGISTRATION_TOPIC_ARN", "")

File: backend/config/environment.js
/**
 * Validated environment settings.
 */

const EVENTS_TABLE = process.env.EVENTS_TABLE || "event-ticketing-events";
const REGISTRATIONS_TABLE = process.env.REGISTRATIONS_TABLE || "event-ticketing-registrations";
const EMAIL_INDEX = process.env.EMAIL_INDEX || "email-index";
const REGISTRATION_ID_INDEX = process.env.REGISTRATION_ID_INDEX || "registration-id-index";
const REGISTRATION_TOPIC_ARN = process.env.REGISTRATION_TOPIC_ARN || "";
const ALLOWED_ORIGIN = process.env.ALLOWED_ORIGIN || "*";

module.exports = {
  EVENTS_TABLE,
  REGISTRATIONS_TABLE,
  EMAIL_INDEX,
  REGISTRATION_ID_INDEX,
  REGISTRATION_TOPIC_ARN,
  ALLOWED_ORIGIN,
};
